src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+
 public class MovieRequestDto {
+
+    @NotNull
     private String title;
+    @NotNull
     private String description;
```

Comment:
I think the description and title can't contain only 1 character, add appropriate annotation